pkg/cmd/factory/default.go

```diff
@@ -1,14 +1,15 @@
 package factory
 
 import (
+	"context"
 	"fmt"
 	"net/http"
 	"os"
 	"regexp"
 	"time"
 
 	"github.com/cli/cli/v2/api"
-	"github.com/cli/cli/v2/context"
+	ghContext "github.com/cli/cli/v2/context"
```

Comment:
Had to rename this in numerous places. We should probably just wholesale change the name of this package as it conflicts with the Go `context` package.

Reply:
Will look into doing this in a follow up PR.